toxav/rtp.c

```diff
@@ -467,7 +467,8 @@ static int handle_rtp_packet(Messenger *m, uint32_t friendnumber, const uint8_t
         return -1;
     }
 
-    if (header.offset_full >= header.data_length_full) {
+    if (header.offset_full >= header.data_length_full
```

Comment:
How about: `if (header.data_length_full != 0 && header.offset_full >= header.data_length_full)`? I.e. either we don't have a full data length, or the offset can't be greater-equal it.